drivers/hmis/app/controllers/hmis/app_settings_controller.rb

```diff
@@ -10,8 +10,24 @@ class Hmis::AppSettingsController < Hmis::BaseController
 
   def show
     okta_enabled = ENV['HMIS_OKTA_CLIENT_ID'].present? && ENV['OKTA_DOMAIN'].present?
+
+    logo = ENV['LOGO']
+    logo_path = if SerializedAsset.exists?(logo)
+      SerializedAsset.get_src(logo)
+    else
+      "theme/logo/#{logo}"
```

Comment:
HMIS may want a different logo than the warehouse, but this seems fine for now (until I manage to get the admin for theming in place.)